glidertest/tools.py

```diff
@@ -52,6 +59,10 @@ def grid2d(x, y, v, xi=1, yi=1):
     XI: x data gridded in x and y space with xi and yi resolution
     YI: y data gridded in x and y space with xi and yi resolution
 
+    Original author
+    ----------------
+    Bastien Queste (https://github.com/search?q=owner%3Abastienqueste%20grid2d&type=code)

```

Comment:
Here you should link to one of the instances of this code. Probably the one in gliderad2cp

https://github.com/bastienqueste/gliderad2cp/blob/de0652f70f4768c228f83480fa7d1d71c00f9449/gliderad2cp/process_adcp.py#L140